### PR TITLE
proj: OptiKey

### PR DESCRIPTION
**Pull request by OscarAblinger**
Note that on the machine that I'm currently using, I don't have yq or yaml-lint installed and it would honestly be a bit of a hassle right now. Could you generate the readme, please?

That also means, that I couldn't test the output. Sorry if there is gonna be problems with it.

### CODE REVIEW

**Comment by OscarAblinger** [comment]
nvm, just saw your new issue

**Comment by AdrienLemaire** [comment]
Thanks @AblingerOscar ! If you don't mind, let's leave this PR open so I can use it as test for #27.

File: src/opti_key.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          OptiKey-logo-600x200.png"
+starsUrl: "https://img.shields.io/github/stars/OptiKey/OptiKey.svg\
+           ?style=social&label=Star&maxAge=2592000"
+numberContributors:
+  url: "https://api.github.com/repos/OptiKey/OptiKey/stats/contributors"
+  format: json  # or yaml or xml
+  accessor: list  # or number or property name prepended by ?
+
+
+ratingComment: Very active and impactful, but no good Contribution guide
```
Comment (AdrienLemaire) [inline src/opti_key.yaml:33]: Maybe we can contact them to improve it? If the project is very active, then maybe the contribution guide isn't so important in their case? We just want to verify that a project is easy to join in the end :)

Comment (OscarAblinger) [reply src/opti_key.yaml:33]: hmm, I'll look for issues addressing this and might open one

Comment (OscarAblinger) [reply src/opti_key.yaml:33]: Oh apparently they estimate to have around 2000 active users as of march. That might be a good info to include…
source: https://github.com/OptiKey/OptiKey/issues/544#issuecomment-469251533

Comment (OscarAblinger) [reply src/opti_key.yaml:33]: Actually I just found https://github.com/OptiKey/OptiKey/wiki/Contribute through googling.
I might still suggest moving that link to the CONTRIBUTING.md at least

Comment (AdrienLemaire) [reply src/opti_key.yaml:33]: Nice catch

Comment (JuliusSweetland) [reply src/opti_key.yaml:33]: @AblingerOscar @Fandekasp I have added a link from the Contributing.md file to the contributing guide. I will look to enhance this guide soon, but first the solution architecture is being overhauled a bit. OptiKey has grown so much that there is now scope to split it into 4 distinct applications (all under the OptiKey umbrella and part of the same solution) which are tailored to the end user's requirements:

Pro - speech, mouse and keyboard control
Chat - full screen speech
Symbol - communication using symbol boards for pre-literate users
Mouse - only mouse control

Comment (OscarAblinger) [reply src/opti_key.yaml:33]: @JuliusSweetland thank you for your elaboration :)

I think I'll update the contribution link to go to the `CONTRIBUTING.md` file again. Once the project is split up, we'd presumably also have to update some links, but for now, I think that the rest is okay.

**Comment by AdrienLemaire** [comment]
Couldn't work on #27 because I had not subscribed to Github actions Beta with my personal account. Merging

**Review by AdrienLemaire — APPROVED** [review]
LGTM

**Pull request merged**